docs/source/cuda-reference/kernel.rst

```diff
@@ -469,10 +469,13 @@ precision parts of the CUDA Toolkit documentation.
 16-bit Floating Point Intrinsics
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. function:: numba.cuda.is_fp16_supported ()
```

Comment:
I did make a comment on #8744 about not wanting to introduce additional uses of the term `fp16`, but I'm a bit torn here because the the module with the intrinsics is named `numba.cuda.fp16`. However, given that a user would normally expect to use operators rather than intrinsics, I think I'd suggest we call it `is_float16_supported()`:

```suggestion
.. function:: numba.cuda.is_float16_supported ()
```